bdb/temptable.c

```diff
@@ -750,15 +748,18 @@ static struct temp_table *bdb_temp_table_create_type(bdb_state_type *bdb_state,
             action = bdb_temp_table_maybe_set_priority_thread(bdb_state);
         }
 
+        int rc = 0;
         switch (action) {
         case TMPTBL_PRIORITY:
-            comdb2_objpool_forcedborrow(bdb_state->temp_table_pool,
-                                        (void **)&table);
+            rc = comdb2_objpool_forcedborrow(bdb_state->temp_table_pool, (void **)&table);
             break;
         case TMPTBL_WAIT:
-            comdb2_objpool_borrow(bdb_state->temp_table_pool, (void **)&table);
+            rc = comdb2_objpool_borrow(bdb_state->temp_table_pool, (void **)&table);
             break;
         }
+        if (rc) {
+            logmsg(LOGMSG_ERROR, "comdb2_objpool_borrow returns rc %d\n", rc);
```

Comment:
Can be either `comdb2_objpool_forcedborrow` or `comdb2_objpool_borrow`. You may also want to include a `strerror()` here.